### PR TITLE
Almet/trim members

### PR DESCRIPTION
**Pull request by almet**
to merge after #366

### CODE REVIEW

File: ihatemoney/forms.py [hunk 1]
```diff
@@ -15,6 +15,10 @@
 from ihatemoney.models import Project, Person
 from ihatemoney.utils import slugify
 
+def strip_filter(string):
+    if hasattr(string, 'strip'):
```
[the remark this answers is not in the document]
Comment (Glandos) [reply ihatemoney/forms.py]: And based on [EAFP](https://docs.python.org/3/glossary.html#term-eafp), it should be written as:
```python
try:
    return string.strip()
except:
    return string
```

[the remark this answers is not in the document]
Comment (almet) [reply ihatemoney/forms.py]: Thanks for the tips :-) I'm still doing defensive programming, is all.

**Comment by almet** [comment]
Rebased. To merge once tests are green.

**Comment by eMerzh** [comment]
hey @almet , this looks good from here ... 
could you rebase it once again, and maybe remove the two unrelated changes (spacings in po and tests) so it might get merged :)

**Comment by eMerzh** [comment]
@almet hehe, bloody lint :p 
aside from that still LGTM ;-)

**Pull request merged**